makefile

```diff
@@ -45,6 +45,7 @@ ifneq ($(filter-out clean clean-% print-% help help-% manual stan-update/% stan-
 endif
 
 CMDSTAN_VERSION := 2.22.0
+CMDSTAN_VERSION_DOC := 2.22
```

Comment:
Not related to this PR, but can I request a fix for the Cmdstan version on develop, the patch version was only applied to master. Line 47 here and https://github.com/stan-dev/cmdstan/blob/develop/.github/ISSUE_TEMPLATE.md are those that require the change. Thanks!

Reply:
I will let Nic review the rest, this is just a request. I can open a separate PR if you would rather do that, no problem.